pygorithm/Backtracking/N-Queen.py

```diff
@@ -0,0 +1,89 @@
+# Python program to solve N Queen 
+# Problem using backtracking 
+
+global N 
+N = 4
+
+def printSolution(board): 

```

Comment:
Change function name to `print_solution` so as follow Python PEP8 styling

Reply:
Done